apps/system/lockscreen/js/widgets/clock/lockscreen_clock_widget.js

```diff
@@ -21,35 +22,36 @@
     return (new LockScreenClockWidgetSetup(this));
   };
 
+  LockScreenClockWidget.prototype.updateFormatters = function() {
+    this.dateFormatter = new Intl.DateTimeFormat(navigator.languages, {
```

Comment:
Sorry but I don't see any line indicates the `Intl` is a global JavaScript object or an API. Since you use it here without any `global` mark or including line, and the bug mention nothing about that.